src/legacy/core_plugins/kibana/public/management/sections/objects/lib/resolve_saved_objects.ts

```diff
@@ -50,7 +63,12 @@ function addJsonFieldToIndexPattern(target, sourceString, fieldName, indexName)
     }
   }
 }
-async function importIndexPattern(doc, indexPatterns, overwriteAll, confirmModalPromise) {
+async function importIndexPattern(
+  doc: SavedObjectsRawDoc,
+  indexPatterns: IndexPatternsContract,
+  overwriteAll: boolean,
+  confirmModalPromise: OverlayStart['openConfirm']
```

Comment:
Is it a method or a promise? Signature says that's a method. Let's rename to something more meaningful - `openConfirm`, for example